pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/PythonExtension.groovy

```diff
@@ -81,6 +84,7 @@ class PythonExtension {
 
     public PythonExtension(Project project) {
         this.details = new PythonDetails(project)
+        this.project = project
```

Comment:
This class does not use the `project` outside the constructor.
It simply passes it to its parts, including the details.
We probably don't need this as a private variable.